application/common/models/Mfa.php

```diff
@@ -80,6 +82,26 @@ public function loadData(string $rpOrigin = '')
         }
     }
 
+    /**
+     * Adds ['id' => MfaWebauthn->id, 'label' => MfaWebauthn->label] to the Mfa's data property
+     *  for each of a webauthn type MFA's MfaWebauthn children
+     */
+    public function loadMfaWebauthns()
+    {
+        if ($this->type != self::TYPE_WEBAUTHN) {
+            return;
+        }
+
+        if (! isset($this->data)) {
+            $this->data = [];
+        }
+
+        $webauthns = MfaWebauthn::findAll(['mfa_id' => $this-> id]);
+        foreach ($webauthns as $webauthn) {
+            $this->data[] = ['id' => $webauthn->id, 'label' => $webauthn->label];
```

Comment:
Yep. I'll add `, 'verified' => 1`

Reply:
Per our discussion, it looks like `mfa_webauthn.verified` is an unnecessary field, since we don't create the record until after we verify the registration attempt.

Reply:
When you move this loop up into `loadData()`, you will want to run `make basemodels` to regenerate our "...Base" model classes. Then there will be a `$mfa->mfaWebauthns` property that is a list of all of those child records (avoiding the need to manually call `findAll()`.